packages/web/src/store/sagas.ts

```diff
@@ -86,6 +86,7 @@ import reachabilitySagas from 'store/reachability/sagas'
 import routingSagas from 'store/routing/sagas'
 import signOutSagas from 'store/sign-out/sagas'
 import tokenDashboardSagas from 'store/token-dashboard/sagas'
+import uncommonAccountSagas from 'store/account/sagas'
```

Comment:
ah, so looks like store/account/sagas merges commonSagas with it, so can just rename this to `accountSagas` and remove the `accountSagas` from common in this file?